bin/env_variables_all.txt

```diff
@@ -139,4 +139,5 @@ USER_PROFILE_S2S_AUTHORISED_SERVICES=ccd_data,ccd_definition,ccd_admin
 XUI_LAUNCH_DARKLY_CLIENT_ID=1
 DB_USE_SSL=false
 ES_ENABLED_DOCKER=false
-IDAM_KEY_CCD_PS=AAAAAAAAAAAAAAAA
\ No newline at end of file
+IDAM_KEY_TS_TRANSLATION_SERVICE=AAAAAAAAAAAAAAAC
+TS_TRANSLATION_SERVICE_IDAM_CLIENT_SECRET=AAAAAAAAAAAAAAAD
```

Comment:
I don't think translation service will be using an IDAM client: so TS_TRANSLATION_SERVICE_IDAM_CLIENT_SECRET can probably be removed.

Reply:
These are not present in WLTS-6, they are being added as part of WLTS-14

Reply:
done